bacon/controller/AddTransactionViewController.swift

```diff
@@ -181,18 +171,29 @@ class AddTransactionViewController: UIViewController {
         timeLabel.text = Constants.getDateLessPreciseFormatter().string(from: dateTime)
     }
 
+    private func displayTags(tags: Set<Tag>) {
+        var tagString = ""
+        for tag in tags {
+            tagString += tag.toString() + "  "
```

Comment:
Is that supposed to be 2 spaces o.o

Reply:
ye haha cos I tried one space it looks crammed